other/materials_designer/specific_examples/passivation_surface_silicon_surface.ipynb

```diff
@@ -0,0 +1,357 @@
+{
```

Comment:
<p>The lines are too wide - let's reformat?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>